i18n/en.json

```diff
@@ -564,8 +564,32 @@
     "translation": "Error to update the current channel."
   },
   {
-    "id": "api.command_channel_header.update_channel.success",
-    "translation": "Channel header successfully updated."
+    "id": "api.command_channel_purpose.name",
+    "translation": "purpose"
+  },
+  {
+    "id": "api.command_channel_purpose.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_purpose.desc",
+    "translation": "Edit the channel purpose."
```

Comment:
Can we remove the period here, so it's consistent with other autocomplete descriptions